packages/truffle-core/lib/command.js

```diff
@@ -100,6 +100,35 @@ class Command {
       }
     });
 
+    //Check unsupported command line flag according to the option list in help
+    try {
+      const inputOptions = [];
+      inputStrings.map(i => {
+        if (i.startsWith("--")) {
```

Comment:
should the `if` statements maintain consistency? If so, which one is preferred, this style and or no `{}` for one liners like you have on [line 115](https://github.com/trufflesuite/truffle/pull/2116/files#diff-8e8c8e19fd9034bcaa495fe10529a3d3R115)?

Reply:
My personal preference is no `{}`s for one-liners. Otherwise I always include them. I'm not sure we have established a strict convention though.

Reply:
I agree with you @eggplantzzz  I like the one-liners without the `{}`. As for convention, I hear ya, I would say that as a rule of thumb, one should stick with the same style within a file :)

Reply:
Thanks for the feedback. I will follow the style on file.